molgenis-data-vcf/src/main/java/org/molgenis/data/vcf/utils/VcfWriterUtils.java

```diff
@@ -75,8 +76,7 @@ public static void writeVcfHeader(File inputVcfFile, BufferedWriter outputVCFWri
 		{
 			outputVCFWriter.close();
 			inputVcfFileScanner.close();
-			throw new MolgenisInvalidFormatException(
-					"Did not find ## on the first line, are you sure it is a VCF file?");
+			throw new InvalidHeaderException();
```

Comment:
This error message is no longer used because ```InvalidHeaderException```'s message is '_Header does not start with #CHROM, are you sure it is a VCF file?_'

Reply:
This comment is processed, new exception was introduced for the other place where this was thrown and the message of this exception is updated